notebooks/work-in-progress/explore_fuel_allocation.ipynb

```diff
@@ -2,8 +2,8 @@
  "cells": [
```

Comment:
<p>I don't have a good sense of what these numbers ought to be, either in isolation, or relative to each other, or what the quantities being compared represent. Are they all supposed to be 100.0%?</p><p> </p><p>Having some indication as to the nature of the tests that have been passed would be helpful.</p><p> </p><p>What's happening to the fuel that's reported in units that contain null values in the boiler fuel table? Is it being counted in some contexts but not in others?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>